### DISCUSSION TITLE
Istio support with Odigos

### DISCUSSION THREAD

**Discussion by miospruit** [odigos-io/odigos#32] [Q&A] [answered]
Hi, I am new to monitoring and observability within microservice architecture. I was wondering, if this awsome project will run on a cluster with istio. This might be a redundant question but i can't seem to find the answer my self.
thanks you all in advance!

**Comment by edeNFed** [comment, marked as answer]
Hi @miospruit 
Yes. There is no problem running Odigos alongside Istio. I tried the Bookinfo example from Istio's docs and it worked great. I will make sure to document this better

**Reply by miospruit** [reply]
Hi @edeNFed,
thats awesome to hear, thanks for the fast response
i'm trying it out today!
